stubs/RPi.GPIO/RPi/GPIO/__init__.pyi

```diff
@@ -0,0 +1,52 @@
+from collections.abc import Callable
+from typing import Any, Literal
+from typing_extensions import TypeAlias
+
+VERSION: str
+RPI_INFO: dict[str, Any]
```

Comment:
This could be a `TypedDict`, but this can always be done later.

Reply:
Thank you for the review @srittau 

I just tried that, and mypy failed with: https://github.com/fperrin/typeshed/actions/runs/7710857960/job/21015067471?pr=3

> mypy --platform win32 --python-version 3.9 on the test cases for 'requests': FAILURE
> ...
>   File "mypy/fixup.py", line 127, in visit_symbol_table
>   File "mypy/lookup.py", line 31, in lookup_fully_qualified
> AssertionError: Cannot find module for resource.struct_rusage

I really doubt this is related to my change, but who knows...

Reply:
@fperrin sorry about that! That's entirely unrelated, and it's very much on our minds right now: see

- #11220

Reply:
in that case I'll ignore that failure and push Sebastian's suggestion